include/deal.II/fe/fe_tools_extrapolate.templates.h

```diff
@@ -121,6 +131,15 @@ namespace FETools
         bool
         operator<(const CellData &rhs) const;
 
+        bool
+        operator==(const CellData &rhs) const
+        {
+          Assert(dof_values == rhs.dof_values, ExcInternalError());
+          Assert(tree_index == rhs.tree_index, ExcInternalError());
+          // Assert (quadrant == rhs.quadrant, ExcInternalError());
```

Comment:
?

Reply:
Yes, this whole function clearly makes no sense :-) It must have been a left-over from some debugging sessions. Thanks for finding it!